nexus/swags/models.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Third Party Stuff
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+# nexus Stuff
+from nexus.base.models import ImageMixin, TimeStampedUUIDModel
+
+
+class Swag(ImageMixin, TimeStampedUUIDModel):
+    item = models.ForeignKey('SwagItem', on_delete=models.CASCADE, verbose_name=_('Item'), null=False, blank=False)
+    users = models.ManyToManyField('users.User', through='SwagOwnership')
+    description = models.TextField(verbose_name=_('Description'), null=False, blank=False)
```

Comment:
We also need to add `image` in this model.

Reply:
Inheriting `ImageMixin` adds an `image` field.